actors/Weapons/Slot3/QUADBARREL.dec

```diff
@@ -2128,7 +2128,7 @@ ACTOR PB_QuadSG : PB_Weapon
 			A_GiveInventory("PB_Shell", 1);
```

Comment:
Found another bug here. Above this line, on the TakeOneByOneAkimboLeft state, replace QSSGAmmoCounter with LeftQSSGAmmoCounter.

Reply:
Thanks for noticing while the request is (temporarily) in the draft stage